google/cloud/firestore_v1/__init__.py

```diff
@@ -97,6 +97,13 @@
 # from .types.write import DocumentDelete
 # from .types.write import DocumentRemove
 from .types.write import DocumentTransform
+from typing import List
+
+__all__: List[str]
+__version__: str
+
+__all__: List[str]
+__version__: str
```

Comment:
Is there a reason for this duplication?

Reply:
nope. This is likely a bug from the generation step of adding these. will remove this one. do flag others. I found I think most, but some may have sneaked by.

Reply:
I believe I flagged all duplicates in my initial review. I was actually worried about duplicate comments about duplication, so I'm happy that that wasn't an issue :)